packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -94,30 +94,39 @@ function useFormTextAnimation() {
   };
 }
 
+type CalculateConfigProps = {
+  recalculate?: string | boolean | undefined;
+};
 type Px = number;
-const useCalculateErrorTextHeight = () => {
+const useCalculateErrorTextHeight = (config: CalculateConfigProps = {}) => {
   const [height, setHeight] = useState<Px>(24);
 
-  const calculateHeight = useCallback((element: HTMLElement | null) => {
-    if (element) {
-      const computedStyles = getComputedStyle(element);
-      const marginTop = parseInt(computedStyles.marginTop.replace('px', ''));
-
-      setHeight(prevHeight => {
-        const newHeight = 1.5 * marginTop + element.scrollHeight;
-        if (prevHeight < newHeight) {
-          return newHeight;
-        }
-        return prevHeight;
-      });
-    }
-  }, []);
+  const calculateHeight = useCallback(
+    (element: HTMLElement | null) => {
+      if (element) {
+        const computedStyles = getComputedStyle(element);
+        const marginTop = parseInt(computedStyles.marginTop.replace('px', ''));
+
+        const newHeight = 1.1 * marginTop + element.scrollHeight;
+        setHeight(newHeight);
+      }
+    },
+    [config?.recalculate],
```

Comment:
My understanding here is that this is used simply to force a re-render. And its value is not actually used.

Would the following work ? 

```jsx
const calculateHeight = useCallback((element: HTMLElement | null, string) => {},[])

<FormInfoText ref={ref =>calculateHeight(ref,informationMessage)} />
<FormErrorText ref={ref => calculateHeight(ref,errorMessage)} />
<FormWarningText ref={ref => calculateHeight(ref,warningMessage)} />

```

Reply:
this makes sense